ref/README.md

```diff
@@ -20,7 +20,7 @@ The following packages are required for building and running this kernel:
 * [git-lfs](https://git-lfs.github.com/)
 
 ## Prerequisites
-This code requires git-lfs. Before proceeding to the build instructions, verify that
```

Comment:
Here's what I see:

This code requires git-lfs. Before cloning the repository, verify that git-lfs is installed. This only needs to be done once per user per machine.

$ git lfs install

But I think we need to clearly tie the "verify that git-lfs is installed" with how to do that.  I would suggest the following:

This code requires git-lfs. Before cloning the repository, verify that git-lfs is installed by issuing the following command.  This command only needs to be done once per user per machine.

$ git lfs install